py_gtfs_rt_ingestion/batch_files.py

```diff
@@ -48,44 +50,18 @@ def parseArgs(args) -> BatchArgs:
         action='store_true',
         help='print out a pretty summary of the batches')
 
-    parser.add_argument(
-        '--output',
-        dest='output',
-        type=str,
-        help='filepath to dump out batches as event json list')
-
-    parsed_args = BatchArgs(**vars(parser.parse_args(args)))
-
-    return parsed_args
-
-def file_list_from_file(input_file: str) -> Iterable[(str, int)]:
-    """
-    take a file that is the output of `aws s3 ls <dir>` and yield out filename,
-    filesize tuples.
-    """
-    for file_info in open(input_file):
-        (date, time, size, filename) = file_info.split()
-        yield (filename, size)
-
-def make_file_list(args: BatchArgs) -> Iterable[(str, int)]:
-    """
-    based on args, yield out filename, filesize tuples either from a local file
-    or from our s3 bucket.
-    """
-    if args.input_file:
-        return file_list_from_file(args.input_file)
+    return vars(parser.parse_args(args))
 
-    if args.s3_prefix:
-        return file_list_from_s3(file_prefix=args.s3_prefix)
+def main(batch_args: BatchArgs) -> None:
```

Comment:
really recommend looking at this diff in Split view.

this main function gets the file names and sizes from s3, passes them to the batching function, logs out a summary of them or prints them to std::out (so you can capture with `./batch_files.py > events.json` if you like). eventually it should trigger the ingestion jobs.